src/media/Icon.jsx

```diff
@@ -5,6 +5,22 @@ import { MEDIA_SIZES } from '../utils/designConstants';
 export const ICON_CLASS = 'svg';
 
 /**
+ * @param {String} shape - icon shape
+ * @param {String} size - icon size
+ * @returns {String} icon name (with or without suffix)
+ */
+export const getIconShape = (shape, size) => {
+
+	// third party icons (yahoo, facebook, etc) do not have small variants
+	if (shape.includes('external')) {
+		return shape;
+	}
+
+	const suffix = size == 'xs' ? '--small' : '';
```

Comment:
At `xs` sizes, we draw icons with a thinner stroke and simplified details. Take a look at `photo-album` here and toggle the "toggle small" checkbox in the top right of the page: https://meetup.github.io/swarm-icons/

Because we render a separate SVG for `xs` sizes, we have to make a case for it in our `Icon` component.